infra/main.bicep

```diff
@@ -858,4 +860,5 @@ output USE_KEY_VAULT bool = useKeyVault
 output AZURE_APP_SERVICE_HOSTING_MODEL string = hostingModel
 output FRONTEND_WEBSITE_NAME string = hostingModel == 'code' ? web.outputs.FRONTEND_API_URI : web_docker.outputs.FRONTEND_API_URI
 output ADMIN_WEBSITE_NAME string = hostingModel == 'code' ? adminweb.outputs.WEBSITE_ADMIN_URI : adminweb_docker.outputs.WEBSITE_ADMIN_URI
+output USE_INTEGRATED_VECTORIZATION bool = use_int_vectorization
```

Comment:
If we are following the pattern I think we need AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION